docs/samples/Microsoft.ML.Samples/Dynamic/LightGBM/LightGBMBinaryClassification.cs

```diff
@@ -0,0 +1,103 @@
+            {
+                Separators = new[] { ',' },
+                HasHeader = true,
+                Columns = new[]
+                {
+                    new TextLoader.Column("age", DataKind.R4, 0),
+                    new TextLoader.Column("workclass", DataKind.Text, 1),
+                    new TextLoader.Column("fnlwgt", DataKind.R4, 2),
+                    new TextLoader.Column("education", DataKind.Text, 3),
+                    new TextLoader.Column("education-num", DataKind.R4, 4),
+                    new TextLoader.Column("marital-status", DataKind.Text, 5),
+                    new TextLoader.Column("occupation", DataKind.Text, 6),
+                    new TextLoader.Column("relationship", DataKind.Text, 7),
+                    new TextLoader.Column("ethnicity", DataKind.Text, 8),
+                    new TextLoader.Column("sex", DataKind.Text, 9),
+                    new TextLoader.Column("capital-gain", DataKind.R4, 10),
+                    new TextLoader.Column("capital-loss", DataKind.R4, 11),
+                    new TextLoader.Column("hours-per-week", DataKind.R4, 12),
+                    new TextLoader.Column("native-country", DataKind.Text, 13),
+                    new TextLoader.Column("Label", DataKind.Bool, 14)
+                }
+            });
+
+            // Read the data, and leave 10% out, so we can use them for testing
+            var (trainData, testData) = mlContext.BinaryClassification.TrainTestSplit(reader, testFraction: 0.1);
+
+            // Create the Estimator
+            var pipeline = mlContext.Transforms.Categorical.OneHotEncoding(new OneHotEncodingEstimator.ColumnInfo[]
+                {
+                    new OneHotEncodingEstimator.ColumnInfo("marital-status"),
+                    new OneHotEncodingEstimator.ColumnInfo("occupation"),
+                    new OneHotEncodingEstimator.ColumnInfo("relationship"),
+                    new OneHotEncodingEstimator.ColumnInfo("ethnicity"),
+                    new OneHotEncodingEstimator.ColumnInfo("sex"),
+                    new OneHotEncodingEstimator.ColumnInfo("native-country"),
+                })
+            .Append(mlContext.Transforms.FeatureSelection.SelectFeaturesBasedOnCount("native-country", count: 10))
+            .Append(mlContext.Transforms.Concatenate("Features",
+                                                        "age",
+                                                        "education-num",
+                                                        "marital-status",
+                                                        "relationship",
+                                                        "ethnicity",
+                                                        "sex",
+                                                        "hours-per-week",
+                                                        "native-country"))
+            .Append(mlContext.Transforms.Normalize("Features"))
```

Comment:
see if you can use from: [https://github.com/dotnet/machinelearning/pull/2483](https://github.com/dotnet/machinelearning/pull/2483)